free5gc-operator/config/crd/bases/workload.nephio.org_imageconfigs.yaml

```diff
@@ -0,0 +1,122 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.11.1
+  creationTimestamp: null
+  name: imageconfigs.workload.nephio.org
+spec:
+  group: workload.nephio.org
```

Comment:
Is this free5gc specific? If so, probably should be in a free5gc group. If not, probably should clean up the descriptions to not talk about free5gc (and probably should move to a different repo...).

Reply:
No, it's not free5gc specific. Which repo is better then for this CR? `nephio-project/api` in the `nf_deployments/v1alpha1` folder?

Reply:
Yes, that sounds right.